docs/index.md

```diff
@@ -10,6 +10,18 @@
 - **Seamless integration with [RDFLib](https://rdflib.readthedocs.io/)**
 - **Stream processing support** for large datasets or streams of many RDF graphs/datasets
 
+## Compatibility
+
+PyJelly is compatible with Python 3.9 and newer. Python 3.9–3.13 are tested in CI and are guaranteed to work.
+
+The following table shows the compatibility of the PyJelly implementation with other libraries:
+
+| PyJelly | Python     | RDFLib         |
+|---------|------------|----------------|
+| latest  | 3.9 – 3.13 | 7.x (>= 7.1.4) |
```

Comment:
1. It's "pyjelly" not "PyJelly". Branding details matter :)
2. Instead of "latest" write down the exact range of versions – 0.1.0–(whatever is the current version). To do this, you need to use a macro, just like we do in Jelly-JVM docs (see their source code). Macros are defined here: https://github.com/Jelly-RDF/pyjelly/blob/main/docs/env.py 

Macro docs: https://mkdocs-macros-plugin.readthedocs.io/en/latest/